packages/server/lib/modes/record.js

```diff
@@ -1026,10 +1037,14 @@ const createRunAndRecordSpecs = (options = {}) => {
           .finally(() => {
             // always attempt to upload stdout
             // even if uploading failed
-            return updateInstanceStdout({
+            const result = updateInstanceStdout({
```

Comment:
`updateInstanceStdout` pushes stdout data to Cy Cloud & returns a promise - should this promise resolve before ending the span?